### PR TITLE
Convert all data variables in a Dataset to the same unit.

### PR DESCRIPTION
**Pull request by mikapfl**
Add the ability to convert all data variables in a Dataset to the same unit by giving a single unit-like instead of a dictionary.

Closes: #45

### CODE REVIEW

File: pint_xarray/accessors.py [hunk 1]
```diff
@@ -634,7 +637,45 @@ def to(self, units=None, **unit_kwargs):
...
+            u        (x) int64 [s] 0 1 2 3 4
+        Dimensions without coordinates: x
+        Data variables:
+            a        (x) float64 [g] 0.0 250.0 500.0 750.0 1e+03
+            b        (x) float64 [g] -0.001 -0.00075 -0.0005 -0.00025 0.0
         """
+        if isinstance(units, (str, pint.Unit)):
+            units = {x: units for x in self.ds.keys()}
+            units.update(unit_kwargs)
+            unit_kwargs = None
```
Comment (keewis) [inline pint_xarray/accessors.py:672]: I think it's better to be as close to `DataArray.pint.to` as possible:
```suggestion
            unit_kwargs.update(
                {name: units for name in self.ds.keys() if name not in unit_kwargs}
            )
            units = None
```

**Review by keewis — COMMENTED** [review]
thanks for the PR, @mikapfl, this looks mostly good to me.

Could you also update the changelog? We currently don't credit the authors, but since you're the first community contributor it might be good to start doing that. The syntax might be something like `xarray`'s:
```
- description (:issue:`nn`, ..., :pull:`nn`).
  By `Contributor <https://github.com/contributor>`_.
```

File: pint_xarray/tests/test_accessors.py [hunk 2]
```diff
@@ -273,3 +273,29 @@ def test_roundtrip_data(self, example_unitless_ds):
...
+    ds = xr.Dataset(data_vars={"a": (["t"], a), "b": (["t"], b)}, coords={"t": t})
+
+    ae = np.linspace(0, 10_000, 21) * unit_registry.g
+    be = np.linspace(0, 20.0 / 1000, 21) * unit_registry.g
+    expected = xr.Dataset(
+        data_vars={"a": (["t"], ae), "b": (["t"], be)}, coords={"t": t}
+    )
+
+    actual = ds.pint.to(units_arg, **units_kwargs)
+    assert conversion.extract_units(actual) == conversion.extract_units(expected)
```
[the remark this answers is not in the document]
Comment (mikapfl) [reply pint_xarray/tests/test_accessors.py:300]: Ah, cool, I played around with the different assert_identical functions, but didn't find assert_units_equal.

**Comment by mikapfl** [comment]
Thanks for the review! I have incorporated your suggestions and added a changelog entry.

**Comment by keewis** [comment]
Great. Thanks again, @mikapfl.

**Pull request merged**